### PR TITLE
add cull-max for hard upper limit for container age

### PR DESCRIPTION
**Pull request by minrk**
This limits how long a container can be used, so that a forgotten browser tab or a user cannot squat a container for too long.

Default is four hours

This doesn't work as intended just yet, because containers are currently added to the proxy at launch, meaning that the timer starts when the container is created, not when it is popped out of the pool.

### CODE REVIEW

**Comment by rgbkrk** [comment]
The neat thing about recycling them in the pool is that they still get replenished. Doesn't help a user when they get a depleted one though.

**Comment by minrk** [comment]
It seems a bit odd that containers are being added to the stale routes list that are just sitting idle in the pool (aside: sad and surprised there's no swimming pool emoji here). Later, these stale routes are filtered to create `unpooled_stale_routes`, which is what's ultimately used for culling. So my log statements that I added here are wrong—being stale is actually meaningless unless the container is *also* out of the pool.

I would like to move the proxy-update to happen at allocation-time, rather than pool-spawn-time, which would make some of this simpler. It would, however, add an extra step to the user-redirect, slowing things down a bit.

**Comment by minrk** [comment]
Just came back to this. It works now, tracking start time in its own dict, rather than putting it in the routing table, since routes are not updated when allocation happens.

File: README.md [hunk 1]
```diff
@@ -80,6 +80,35 @@ docker run -d \
 ```
 Usage: orchestrate.py [OPTIONS]
 
+Options:
+
+  --help                           show this help information
+
+/Users/benjaminrk/dev/py/tornado/tornado/log.py options:
```
Comment (rgbkrk) [inline README.md]: May want to trim this out.

Comment (minrk) [inline README.md]: blomp. Fixed, thanks.

**Comment by minrk** [comment]
Thanks! This might help with the zombie containers I've been seeing, where images still have 10 occupied containers even after being off the redirector for two days.

**Pull request merged**